README.md

```diff
@@ -708,6 +708,28 @@ Translations of the guide are available in the following languages:
   a = *(1..3)
   ```
 
+* <a name="trailing-underscore-variables"></a>
+  Avoid the use of unnecessary trailing underscore variables during
+  parallel assignment. Trailing underscore variables are necessary
+  when there is a splat variable defined in the argument chain, and
```

Comment:
in the argument chain -> on the left side of the assignment (sounds better to me)